client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -33,17 +30,14 @@ var MOCK_SITE = {
 	options: {}
 };
 
-var expect = chai.expect;
-
-chai.use( sinonChai );
-
 describe( 'EditorGroundControl', function() {
+	let EditorGroundControl;
+
+	useMockery();
+	useFakeDom();
 	this.timeout( 10 * 1000 );
```

Comment:
>can we drop the explicit timeout? 

Not sure why it was added in the first place, but there's not really any performance optimizations made as part of this migration, so I'd be a bit wary about removing it.